src/common/POLFields/FieldHolding/useHoldings.js

```diff
@@ -0,0 +1,20 @@
+import { useQuery } from 'react-query';
+
+import { useOkapiKy } from '@folio/stripes/core';
+import { LIMIT_MAX } from '@folio/stripes-acq-components';
+
+export const useHoldings = (instanceId) => {
+  const ky = useOkapiKy();
+  const searchParams = { query: `instanceId==${instanceId}`, limit: LIMIT_MAX };
+
+  const { isLoading: isHoldingsLoading, data = {} } = useQuery(
+    ['poLine-instance-holdings', instanceId],
```

Comment:
```suggestion
    ['holdings-storage/holdings', searchParams],
```
Do we need to key this request exclusively to "poLine"?
Basically it's just a request for Instance's Holdings, which can be potentially made in another places too. So we could enjoy the benefits of common react-query cache.